docs/getting_started/important_concepts.md

```diff
@@ -0,0 +1,110 @@
+---
+icon: material/lightbulb
+---
+
+# **Important Concepts**
+
+In this document, we discuss and disambiguate a number of concepts that are central to working with OmniGibson and BEHAVIOR-1K.
+
+## **BEHAVIOR concepts**
+
+At a high level, the BEHAVIOR dataset consists of tasks, synsets, categories, objects and substances. These are all interconnected and are used to define and simulate household robotics.
+
+### Tasks
```

Comment:
this is sorta repetitive wrt @ChengshuLi 's info in his Knowledgebase tutorial...not sure how to merge the two